demos/crossroad_camera_demo/cpp/detection_person_attr.hpp

```diff
@@ -0,0 +1,202 @@
+// Copyright (C) 2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <string>
+
+#include "openvino/openvino.hpp"
+
+#include "gflags/gflags.h"
+#include "utils/slog.hpp"
+#include "detection_base.hpp"
+
+using namespace ov::preprocess;
```

Comment:
You can't use `using namespace` in headers. Including such header implicitly exposes the namespace for the includer. This can be undesired.

Reply:
removed